libraries/botbuilder-core/botbuilder/core/bot_framework_adapter.py

```diff
@@ -157,6 +157,7 @@ def __init__(self, settings: BotFrameworkAdapterSettings):
         self,
```

Comment:
This function should set the type and name of the event activity used to warm up the middleware.

Reply:
Updated.